polyfill/Reality.js

```diff
@@ -108,6 +108,14 @@ export default class Reality extends EventHandlerBase {
 		throw new Error('Exending classes should implement _getLightAmbientIntensity')
 	}
 
+    _getWorldMap() {
+        throw new Error('Exending classes should implement _getWorldMap')
```

Comment:
Exending => Extending

Reply:
Thanks for noticing ... there were a bunch of them, that's what happens when a typo gets copied!  Will upload a fix with the next commit to develop.